shared/modules/persistence_module.go

```diff
@@ -11,14 +11,8 @@ type PersistenceModule interface {
 	Module
 	NewRWContext(height int64) (PersistenceRWContext, error)
```

Comment:
If we have a `GetWriteContext`, can we also rename this to `NewWriteContext`?

Question: Since there's only ever one, does it need a height?